LLaMA_LoRA.ipynb

```diff
@@ -60,20 +60,15 @@
         "# @title A small workaround { display-mode: \"form\" }\n",
         "# @markdown Don't panic if you see an error here. Just click the `RESTART RUNTIME` button in the output below, then Run All again.\n",
         "# @markdown The error will disappear on the next run.\n",
-        "!pip install Pillow==9.3.0 numpy==1.23.5\n",
+        "%pip install Pillow==9.3.0 numpy==1.23.5\n",
```

Comment:
colab seems to understand this slightly better, if nothing else syntax highlighting-wise

Reply:
Hmm, I still can't tell what's the difference here. Can you give me some examples of references if you have time?

All I know about the difference between `%` and `!` is that changing the directory with `%cd` will be preserved, while `!cd` will not.

Reply:
My understanding is that `%pip` is more "native", my friend said:
<img width="721" alt="image" src="https://user-images.githubusercontent.com/1841944/232750993-39260321-a2a5-4bc4-9a57-59ee3ae528c1.png">